yarn.lock

```diff
@@ -719,6 +719,11 @@
     graphql "^15.0.0"
     graphql-tag "^2.10.3"
 
```

Comment:
I am unfamiliar with yarn, so if I messed something up here please let me know. This diff came from:
```
yarn remove @octokit/webhooks
yarn add -D @octokit/webhooks-types@latest
```